### PR TITLE
Add photos for Cody Vuelette

### PR DESCRIPTION
**Pull request by cookieguru**

### CODE REVIEW

File: source/_elephpants/2019-05-23-pink-shopware.md [hunk 1]
```diff
@@ -6,5 +6,15 @@ tags:
     - pink
```
Comment (philipsharp) [inline source/_elephpants/2019-05-23-pink-shopware.md:6]: Looks like this elephpant is actually purple?

**Comment by doppynl** [comment]
I think we run into a color perception issue here. As colors are often perceived differently by different people. Especially if the lighting/backgrounds are different. Remember the gold/blue dress thingy?

Short answer: I would call it violet.

When I use an app to scan the color for this elephpant, it comes back as "violet", which makes sense to me when I look at the name.

Compared to the heroku (which also scans as violet) and is listed here as purple, it is definitely darker. But I would consider the heroku violet, or otherwise pink, but not purple.

I would consider the cody vuelette elephpant violet, and if you don't want to use violet it would be purple. But given the color teal is used, violet is not strange to start using and is to me the most appropriate.



Also: The CakeFest elephpant is listed as blue. to me it is definitely not blue. It should probably be teal or green. So this isn't just a problem for this elephpant.

**Comment by philipsharp** [comment]
Good point--if we have teal, why not violet? I think we can agree, though, that _this_ elephpant is **not** pink. 😃 

I'm going to be doing a review of all the elephants to make sure we haven't drifted too far in wording and categorization. I'll take care of this then.

**Comment by cookieguru** [comment]
@philipsharp I'll stay out of the color identification since I'm colorblind, but I can offer this properly color-balanced photo comparing a few:

2021-04-28 IMG_1827

| Heroku         | OpenGoodies Blue | PHPWomen   | Original Pink |
|----------------|------------------|------------|---------------|
| PHP Roundtable | OpenGoodies Pink | Shopware 6 | AFUP (blue)   |

**Comment by jgrossi** [comment]
for me it's purple, and it's the same colour as the "merge" button background here in GitHub 😂

**Pull request merged**